app/controllers/page_views_controller.rb

```diff
@@ -164,9 +164,10 @@ def update
   end
 
   # @API List user page views
-  # Return the user's page view history in json format, similar to the
-  # available CSV download. Pagination is used as described in API basics
-  # section. Page views are returned in descending order, newest to oldest.
+  # Return a paginated list of the user's page view history in json format,
+  # similar to the available CSV download. Pagination is used as described in
+  # API basics section. Page views are returned in descending order, newest to
+  # oldest.
```

Comment:
This paragraph did already mention pagination as well, but for consistency, I think we should just remove that original sentence (the "Pagination is used as described in API basics section" part), no need to mention where you can find docs on it.